src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -81,17 +84,135 @@
+                                    <ComboBox Grid.Row="1" 
+                                              Width="213"
+                                              HorizontalAlignment="Left"
+                                              ItemsSource="{Binding Path=LanguagesList}"
+                                              SelectedItem="{Binding Path=SelectedLanguage}"
+                                              Style="{StaticResource NoBordersComboBoxStyle}">
+                                    </ComboBox>
+                                </StackPanel>
+                            </Grid>
+
+                            <!--This Grid contains controls for selecting Run Settings options-->
+                            <Grid Grid.Row="1">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="*" />
+                                </Grid.RowDefinitions>
+                                <Label Grid.Row="0"
+                                   Content="Default Run Settings" 
+                                   Foreground="{StaticResource PreferencesWindowFontColor}"/>
+                                <Grid Grid.Row="1">
+                                    <Grid.ColumnDefinitions>
+                                        <ColumnDefinition Width="Auto"/>
+                                        <ColumnDefinition Width="Auto"/>
+                                    </Grid.ColumnDefinitions>
+                                    <RadioButton Grid.Column="0" 
+                                                 MinWidth="80"
+                                                 Style="{StaticResource RunSettingsRadioButtons}"
+                                                 IsChecked="{Binding Path=RunSettingsIsChecked}"
+                                                 Content="Manual"/>
+                                    <RadioButton Grid.Column="1" 
+                                                 MinWidth="80"
+                                                 Style="{StaticResource RunSettingsRadioButtons}"
+                                                 Content="Automatic"/>
+                                </Grid>
+                            </Grid>
+
+                            <!--This Grid contains controls for selecting Font Size-->
+                            <Grid Grid.Row="2">
+                                <Grid.RowDefinitions>
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                    <RowDefinition Height="Auto" />
+                                </Grid.RowDefinitions>
+                                <Label Name="FontSize"
+                                   Grid.Row="0"
+                                   Content="Node Font Size" 
```

Comment:
Same here

Reply:
@QilongTang 
I searched in the resources files but we don't have a similar entry, then I added a new resource entry, see the changes in the commit: https://github.com/DynamoDS/Dynamo/pull/11542/commits/9035f3d2b7de82a15cb6098d2a56a00c1b06836d